docs/commands.md

```diff
@@ -229,15 +229,17 @@ Note: Whenever filters need to be provided, a minimum of one `l=v` filter must b
 Query a range in forward or reverse directions.
 
 ```sql
-TS.RANGE key fromTimestamp toTimestamp [COUNT count] [AGGREGATION aggregationType timeBucket]
-TS.REVRANGE key fromTimestamp toTimestamp [COUNT count] [AGGREGATION aggregationType timeBucket]
+TS.RANGE key fromTimestamp toTimestamp [COUNT count] [OFFSET offset] [AGGREGATION aggregationType timeBucket]
+TS.REVRANGE key fromTimestamp toTimestamp [COUNT count] [OFFSET offset] [AGGREGATION aggregationType timeBucket]
 ```
 
 - key - Key name for timeseries
 - fromTimestamp - Start timestamp for the range query. `-` can be used to express the minimum possible timestamp (0).
 - toTimestamp - End timestamp for range query, `+` can be used to express the maximum possible timestamp.
 
 Optional args:
+* count - Maximum number of returned results per time-series.
+* offset - Offset the timestamps being returned; allows for the starting value to be set.
```

Comment:
We also want to mention this is millisecond based offset to be aligned with other input/output related to timestamps